src/renderer/atlas/custom_shader_ps.hlsl

```diff
@@ -0,0 +1,82 @@
+// The original retro pixel shader

```

Comment:
What's up with this copy? Can it be contributed back to the other copy since it was made to build without warnings?

Do users have to rewrite their shaders when Atlas comes on line?

Reply:
We used to have 2 copies of this shader and I contributed it back to the one in the samples directory. Do you want me to make the change for DxEngine as well? I figured it wouldn't make a difference for it...

> Do users have to rewrite their shaders when Atlas comes on line?

Some users might. Out of all shaders we have only the retro shader produces warnings (which are now treated as errors).